src/search_algo.hpp

```diff
@@ -889,7 +1096,7 @@ _performAlignment(TDepSetH & depSetH,
     unsigned constexpr sizeBatch = seqan::LENGTH<TSimdAlign>::VALUE;
     unsigned const      fullSize = sizeBatch * ((seqan::length(blastMatches) + sizeBatch - 1) / sizeBatch);
 
-    TSimdScore simdScoringScheme(lH.gH.scoringSchemeAlign);
+    TSimdScore simdScoringScheme(lH.gH.scoringSchemeAlignFwd);
```

Comment:
Why are there two versions of the scoringSchemeAlign? Isn't that only used in Alignment (which happens on unreduced seqs)?